virttest/qemu_vm.py

```diff
@@ -1379,7 +1379,7 @@ def sort_key(dev):
         support_cpu_model = decode_to_text(process.system_output("%s -cpu \\?" % qemu_binary,
                                                                  verbose=False,
                                                                  ignore_status=True,
-                                                                 shell=True))
+                                                                 shell=True), encoding='utf-8')
```

Comment:
IMHO, this is just a workaround, because there is some garbage character in the output not unicode characters.

I have mentioned the issue in https://github.com/avocado-framework/avocado/issues/2561, please have a check, thanks.